fastserde/avro-fastserde/src/main/java/com/linkedin/avro/fastserde/SchemaAssistant.java

```diff
@@ -444,6 +464,61 @@ public JClass findStringClass(Schema schema) {
     return outputClass;
   }
 
+  boolean logicalTypeEnabled(Schema schema) {
+//    return modelData != null && schema != null && Utils.isLogicalTypeSupported() && schema.getLogicalType() != null;
+    // TODO uuid in 1.10 no supported ??
```

Comment:
Hmm, weird... I see UUIDs mentioned in the spec as far back as 1.9... but maybe in the actual implementation it's not properly supported until 1.11? IDK... this is the kind of minute detail @radai-rosenblatt would know :D ...

Reply:
Ah yes, you reminded me something.
I did an experiment to see how generated Avro classes change when avro version is downgraded:
- https://github.com/krisso-rtb/avro-logical-types-generated-classes/commits/master
This particular case with `UUID` is caught by commit `Avro version downgraded from 1.11.1 to 1.11.0`
- https://github.com/krisso-rtb/avro-logical-types-generated-classes/commit/81e0be0e222989f81a0fe7461f173084f927aea2
Basically we have:
```
private java.util.UUID uuidField;   // avro 1.11.1
private java.lang.String uuidField; // avro 1.11.0
```
from the same `.avsc` file.
(just take a look at diff for `FastSerdeLogicalTypesTest1.java`)